tpcc/load.go

```diff
@@ -98,7 +120,7 @@ s_dist_07, s_dist_08, s_dist_09, s_dist_10, s_ytd, s_order_cnt, s_remote_cnt, s_
 		sRemoteCnt := 0
 		sData := randOriginalString(s.R, s.Buf)
 
-		v := fmt.Sprintf(`(%d, %d, %d, '%s', '%s', '%s', '%s', '%s', '%s', '%s', '%s', '%s', '%s', %d, %d, %d, '%s')`,
+		v := fmt.Sprintf(`%d,  %d,  %d,  '%s',  '%s',  '%s',  '%s',  '%s',  '%s',  '%s',  '%s',  '%s',  '%s',  %d,  %d,  %d,  '%s'`,
```

Comment:
Using two spaces is ugly. It is just for solving the case that uuid also contains `, `. Currently I don't have any other way to solve this

Reply:
why uuid contains `,`

Reply:
My bad. The original sql contains `unhex(replace(uuid(), '-', ''))`, which needs to decode the uuid. So the result may contains `, `

Reply:
Not needed now.